tfhe/src/lib.rs

```diff
@@ -2,6 +2,57 @@
 //!
 //! TFHE-rs is a fully homomorphic encryption (FHE) library that implements Zama's variant of TFHE.
 
+#![warn(clippy::pedantic)]
+#![allow(clippy::must_use_candidate)]
```

Comment:
erm, no ?

Reply:
After checking the pedantic group looks like we can't do much about this :| so I guess would be the way to go